strapi/strapi
# feat: add addEditViewSidePanel API to strapi

fix: don't pass all the query params to the create route



### What does it do?

* removes erroneous classes, we can reimplement if we find a real reason to add them (not just an add command)
* adds a fake CM plugin (anticipates what the CM will look like at the end of V5 dev)
* adds addEditViewSidePanel API
* adds a way to render Description APIs

### How to test it?

* Go to edit-view
* You should see the default panel render (review-workflows _probably_ won't work, but it should render if the CT has RW enabled & you have EE enabled)

### Related issue(s)/PR(s)

* resolves CONTENT-2015
